### PR TITLE
v10.2.0 - Adapting the default task so that it works with Gulp v4.

### PR DESCRIPTION
**Pull request by SachaZvetelman**
v10.2.0
------------------------------
*May 5, 2020*

## Changed
- Adapted the `default` task so that it works with Gulp v4.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v10.2.0
```
Comment (ashleynolan) [inline CHANGELOG.md:6]: Need to bump the version in `package.json` too

File: config.js [hunk 2]
```diff
@@ -47,6 +47,7 @@ const ConfigOptions = () => {
             },
             jsDir: 'js',
             lintPaths: [],
+            allowEmpty: true,
```
Comment (ashleynolan) [inline config.js:50]: Can you make sure you add this config option to the `README`

File: CHANGELOG.md (hunk 1 above)
Comment (SachaZvetelman) [reply CHANGELOG.md:6]: Oops, good point. I always forget to do it and end up realising when I run `npm publish` 🥇

**Comment by SachaZvetelman** [comment]
I've made the requested changes. Not sure what you guys mean with "`package.json` file is missing"?

**Comment by kevinrodrigues** [comment]
> I've made the requested changes. Not sure what you guys mean with "`package.json` file is missing"?

I meant that the change to the package.json file (bumping the version) wasn't added to the pr :)

**Pull request merged**